p2p/src/error.rs

```diff
@@ -0,0 +1,20 @@
+//! Error types
+
+pub use eyre::{Report, Result};
```

Comment:
Why are these re-exported here?

Reply:
I guess so others could use `crate::error::Report` instead of `eyre::Report`, but we're not doing it => removed. Thanks